src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1522,7 +1523,8 @@ internal static bool ReportInvalidScopedOverrideAsError(MethodSymbol baseMethod,
             {
                 var baseParameter = baseParameters[i];
                 var overrideParameter = overrideParameters[i + overrideParameterOffset];
-                if ((baseParameter.RefKind, overrideParameter.RefKind) is (RefKind.RefReadOnlyParameter, RefKind.In) or (RefKind.In, RefKind.RefReadOnlyParameter))
+                if ((baseParameter.RefKind, overrideParameter.RefKind) is (RefKind.RefReadOnlyParameter, RefKind.In) or (RefKind.In, RefKind.RefReadOnlyParameter) ||
+                    (methodGroupConversion && OverloadResolution.IsRefMismatchAcceptableForMethodConversion(baseParameter.RefKind, overrideParameter.RefKind, compilationOpt: null)))
```

Comment:
>compilationOpt: null [](http://example.com/codeflow?start=158&length=20)

Why is it Ok to not pass compilation?

Reply:
Because the compilation is used only to determine if `ref`/`in` mismatch is allowed, but we get here only if that's true, otherwise the conversion would fail and we would never get to this point to report warnings about the conversion.